### DISCUSSION TITLE
How can a user block access to index.svelete? (routify v3)

### DISCUSSION THREAD

**Discussion by iCodePoet** [roxiness/routify#418] [Q&A]
How can a user block access to index.svelete?

I'm using _module.svelte. If I have the following file structure, an empty page will be loaded due to  _module.svelte despite the absence of index.svelte files when I approach /featureed. What should I do to prevent this?

```
 routes
│   │   │   ├── featured
│   │   │   │   ├── artist.svelte
│   │   │   │   └── search.svelte
│   │   │   └── _module.svelte
│   │   └── _module.svelte
```

version
"@roxi/routify": "^3.0.0-next.69"

**Comment by jakobrosenberg** [comment]
You can create an index.svelte and have it redirect to another page.
